### ISSUE TITLE
Error prepatch 

### ISSUE THREAD

**Issue by gaveer** [Ironii/iKeystones#2]
```1x iKeystones\iKeystones-1.913.lua:235: attempt to call field '?' (a nil value)
[string "@iKeystones\iKeystones-1.913.lua"]:235: in function 'getResiduum'
[string "@iKeystones\iKeystones-1.913.lua"]:1125: in function 'addToTooltip'
[string "@iKeystones\iKeystones-1.913.lua"]:1331: in function <iKeystones\iKeystones.lua:1325>
[string "=[C]:"]: ?
[string "=[C]:"]: ?
[string "=[C]:"]: ?
[string "=[C]:"]: ?
[string "=[C]:"]: ?
[string "=[C]:"]: ?
[string "=[C]:"]: ?
[string "=[C]:"]: ?
[string "=[C]:"]: in function 'SetBagItem'
[string "@FrameXML\ContainerFrame.lua"]:1321: in function 'ContainerFrameItemButton_OnUpdate'
[string "@FrameXML\ContainerFrame.lua"]:1368: in function 'ContainerFrameItemButton_OnEnter'
[string "@Bagnon\common\Wildpants\classes\item.lua"]:363: in function 'ShowTooltip'
[string "@Bagnon\common\Wildpants\classes\item.lua"]:334: in function 'UpdateTooltip'
[string "@Bagnon\common\Wildpants\classes\item.lua"]:174: in function <...face\Bagnon\common\Wildpants\classes\item.lua:173>```

**Comment by Ironii** [comment]
should be fixed with 1.914

**Comment by gaveer** [comment]
ty 💯
